gridstatus/ercot.py

```diff
@@ -1717,8 +1715,17 @@ def _handle_hourly_resource_outage_capacity(self, doc, verbose=False):
         df = self.read_doc(doc, parse=False, verbose=verbose)
         # there is no DST flag column
         # and the data set ignores DST
-        # so, we will default to assuming it is DST
-        df = self.parse_doc(df, dst_ambiguous_default=True, verbose=verbose)
+        # so, we will default to assuming it is DST. We will also
```

Comment:
This dataset has no sense of DST so I don't know how else to handle non-existent times other than dropping them.

https://www.ercot.com/mp/data-products/data-product-details?id=NP3-233-CD